frontend/.dockerignore

```diff
@@ -0,0 +1,2 @@
+.dockerignore
+Dockerfile
```

Comment:
lots of missing empty line end of file, but that's also my fault, did not fixed them I my previous PR

would you mind checking this ?